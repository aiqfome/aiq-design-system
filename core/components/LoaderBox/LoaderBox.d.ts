import React from 'react';
export declare const LoaderBox: React.FC;
//# sourceMappingURL=LoaderBox.d.ts.map