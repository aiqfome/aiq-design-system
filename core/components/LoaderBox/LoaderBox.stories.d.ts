import React from 'react';
declare const _default: {
    component: React.FC<{}>;
    title: string;
};
export default _default;
export declare const basic: React.FC;
//# sourceMappingURL=LoaderBox.stories.d.ts.map