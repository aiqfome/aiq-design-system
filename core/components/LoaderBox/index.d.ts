export * from './LoaderBox';
//# sourceMappingURL=index.d.ts.map