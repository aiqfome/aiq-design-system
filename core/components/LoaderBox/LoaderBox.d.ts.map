{"version":3,"file":"LoaderBox.d.ts","sourceRoot":"","sources":["../../../src/components/LoaderBox/LoaderBox.tsx"],"names":[],"mappings":"AAAA,OAAO,KAAK,MAAM,OAAO,CAAA;AAKzB,eAAO,MAAM,SAAS,EAAE,KAAK,CAAC,EAI7B,CAAA"}