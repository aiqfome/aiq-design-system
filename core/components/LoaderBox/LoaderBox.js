"use strict";
var __rest = (this && this.__rest) || function (s, e) {
    var t = {};
    for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p) && e.indexOf(p) < 0)
        t[p] = s[p];
    if (s != null && typeof Object.getOwnPropertySymbols === "function")
        for (var i = 0, p = Object.getOwnPropertySymbols(s); i < p.length; i++) {
            if (e.indexOf(p[i]) < 0 && Object.prototype.propertyIsEnumerable.call(s, p[i]))
                t[p[i]] = s[p[i]];
        }
    return t;
};
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.LoaderBox = void 0;
const react_1 = __importDefault(require("react"));
const Flex_1 = require("../Flex");
const Loading_1 = require("../Loading");
exports.LoaderBox = (_a) => {
    var props = __rest(_a, []);
    return (react_1.default.createElement(Flex_1.Flex, Object.assign({ justifyContent: 'center', alignItems: 'center', height: '100vh' }, props),
        react_1.default.createElement(Loading_1.Loading, null)));
};
