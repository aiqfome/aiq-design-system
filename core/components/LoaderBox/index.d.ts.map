{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../../src/components/LoaderBox/index.tsx"],"names":[],"mappings":"AAAA,cAAc,aAAa,CAAA"}