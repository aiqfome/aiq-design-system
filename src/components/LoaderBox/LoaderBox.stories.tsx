import React from 'react'

import { LoaderBox } from './LoaderBox'

export default {
  component: LoaderBox,
  title: 'LoaderBox'
}

export const basic: React.FC = () => <LoaderBox />
