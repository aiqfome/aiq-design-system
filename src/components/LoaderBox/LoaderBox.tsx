import React from 'react'

import { Flex } from '../Flex'
import { Loading } from '../Loading'

export const LoaderBox: React.FC = ({ ...props }) => (
  <Flex justifyContent='center' alignItems='center' height='100vh' {...props}>
    <Loading />
  </Flex>
)
