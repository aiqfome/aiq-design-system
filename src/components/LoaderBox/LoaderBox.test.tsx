import React from 'react'

import { LoaderBox } from '../LoaderBox'

import { render } from '../utils/test/render'

describe('must match with the previous snapshot', () => {
  test('snapshot renders', () => {
    const component = render(<LoaderBox />)

    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
