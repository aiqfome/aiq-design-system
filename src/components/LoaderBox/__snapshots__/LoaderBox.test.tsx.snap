// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`must match with the previous snapshot snapshot renders 1`] = `
<div
  className="sc-bdnylx gfFqSG"
  height="100vh"
>
  <svg
    className="sc-gtssRu cJlNhZ"
    size="medium"
    viewBox="25 25 50 50"
  >
    <circle
      cx="50"
      cy="50"
      r="20"
    />
  </svg>
</div>
`;
